{
  final Map<String,Class<? extends Exception>> handlerErrors=new HashMap<>(1);
  handlerErrors.put("TestCaseAuthenticationHander",LoginException.class);
  when(this.casMock.createTicketGrantingTicket(any(Credential.class))).thenThrow(new AuthenticationException(handlerErrors));
}
